src/lib.ts

```diff
@@ -77,6 +79,7 @@ const Components: { [key: string]: any } = {
   NewFlowComponent,
```

Comment:
Why are we globally registering other components under these names?
For example, why not `NewFlow: NewFlowComponent`?
(If the library users will be overriding them in the embedding apps, wouldn't they be registering them under the names they have in their Vue files, e.g. `NewFlow`?)

Reply:
Good point and now I can't think why. Let's make the change.